README.md

```diff
@@ -1,2 +1,37 @@
-# README
-This will be edited later.
+# Ruby on Rails Facebook application
+
+This is the Final Poject in The Odin Project’s Ruby on Rails Curriculum. For more information click --> [Facebook-Odin]
+(https://www.theodinproject.com/courses/ruby-on-rails/lessons/final-project)
+
+## Getting started
```

Comment:
Maybe you can add a bit more on database configuration, what to name database or what file to change for custom name of database etc.
[OPTIONAL]